logstore/store_test.go

```diff
@@ -0,0 +1,59 @@
+package logstore
+
+import (
+	"bytes"
+	"os"
+	"testing"
+)
+
+func initWorkdir() (string, error) {
+	return os.MkdirTemp("", "")
+}
+
+func teardown(path string) {
+	os.RemoveAll(path)
+}
+
+func TestStore(t *testing.T) {
+	dir, err := initWorkdir()
```

Comment:
Again, would be good to have some more tests, especially around records with long values, and a high number of records